lib/elixir/lib/module/types/unify.ex

```diff
@@ -830,8 +830,8 @@ defmodule Module.Types.Unify do
     inspect(literal)
   end
 
-  def format_type({:var, index}, _simplify?) do
-    "var#{index}"
+  def format_type({:var, index}, _simplify?) when is_integer(index) do
```

Comment:
I am not sure if there is a case where the second element of the tuple will not be an index.

Reply:
It will always be an integer.

```suggestion
  def format_type({:var, index}, _simplify?) do
```